docs/how-to/field-configuration.md

```diff
@@ -1,5 +1,8 @@
 # Configuring translatable fields
 
+  ***Note:
+  It is not currently possible to configure StructBlock translatable fields. See [Issue #307](https://github.com/wagtail/wagtail-localize/issues/307) for more details.***
```

Comment:
Best to use https://squidfunk.github.io/mkdocs-material/reference/admonitions/
e.g. 

```suggestion
!!! attention
    It is not currently possible to configure `StructBlock` translatable fields. See [Issue #307](https://github.com/wagtail/wagtail-localize/issues/307) for more details.
```